Resources/Prototypes/Entities/Objects/Tools/flashlights.yml

```
@@ -133,3 +133,29 @@
     slots:
       cell_slot:
         name: power-cell-slot-component-slot-name-default
+
+- type: entity
+  parent: FlashlightLantern
+  id: EmpFlashlight
```

```suggestion
  id: FlashlightEmp
```

Сначала надо писать базовое название, потом что-то новое

---

Исправил.